arcade/physics_engines.py

```diff
@@ -14,6 +14,10 @@
 )
 from arcade.math import get_distance
 
+__all__ = [
+    "PhysicsEngineSimple",
+    "PhysicsEnginePlatformer"
+]
 
 def _circular_check(player: Sprite, walls: List[SpriteList]):
```

Comment:
Top-level module members.
```suggestion
__all__ = [
    "PhysicsEngineSimple",
    "PhysicsEnginePlatformer"
]


def _circular_check(player: Sprite, walls: List[SpriteList]):
```